doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+:banner: banners/extract_api.png
+:types: api
+
+:code-column:
+
+===============
+Extract API
+===============
+
+Odoo provides a service allowing you to automate the processing of your invoices. The service scans your document using an Optical Character Recognition engine (OCR) and then uses AI based algorithms to extract the fields of interest as the total, the due date or even the invoice lines. More information can be found on the `demo page <https://www.odoo.com/page/invoice-automation>`_.
+
+This service is a paid service. Each invoice processing will cost you one credit. Three different sized packs can be bought on `iap.odoo.com <https://iap.odoo.com/iap/in-app-services/259?sortby=date>`_.
+
+You can either use this service directly in the Odoo Accounting App or by using the API. The Extract API which is detailed in the next section allows you to integrate our service directly into your own projects.
+
+Invoices
+========
+The extract API can be contacted at the following JSON-RPC2_ endpoint: `https://iap-extract.odoo.com`
+
+Expected successful flow
+------------------------
+
+1. Call :ref:`/iap/invoice_extract/parse <webservices/extract_api/invoice_parse>` to submit your invoices (one call for each invoice). On success, you receive a `document_id` in the response. 
```

Comment:
Because the route is the same as the section title I guess. Up to you to see if the "(POST)" adds confusion or not, and thus if a label is required or not.

Reply:
Because in that case it *should* directly use the section name / label (unless I'm misremembering), so there's less stuff in the document and both should stay in sync.